mars/tensor/linalg/norm.py

```diff
@@ -299,14 +300,15 @@ def norm(x, ord=None, axis=None, keepdims=False):
 
     """
     x = astensor(x)
+    ndim = x.ndim
 
     if ord == 'fro':
         ord = None
     if axis is not None:
         if isinstance(axis, Iterable):
-            axis = tuple(axis)
+            axis = tuple([validate_axis(ndim, a) for a in axis])
```

Comment:
Could be just `tuple(validate_axis(ndim, a) for a in axis)`. No need to convert to list first.

Reply:
Thank you. I learn it.